src/Features/Core/Portable/Completion/Providers/AbstractOverrideCompletionProvider.cs

```diff
@@ -37,6 +38,20 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
 
         protected override Task<ISymbol> GenerateMemberAsync(ISymbol newOverriddenMember, INamedTypeSymbol newContainingType, Document newDocument, CompletionItem completionItem, CancellationToken cancellationToken)
         {
+            // Special case: if you are overriding object.ToString(), we will make the return value as non-nullable. The return was made nullable because
+            // are implementations out there that will return null, but that's not something we really want new implementations doing. We may need to consider
+            // expanding this behavior to other methods in the future; if that is the case then we would want there to be an attribute on the return type
+            // rather than updating this list, but for now there is no such attribute until we find more cases for it. See
+            // https://github.com/dotnet/roslyn/issues/30317 for some additional conversation about this design decision.
+            //
+            // We don't check if methodSymbol.ContainingType is object, in case you're overriding something that is itself an override
+            if (newOverriddenMember is IMethodSymbol methodSymbol &&
+                methodSymbol.Name == "ToString" &&
```

Comment:
`nameof(ToString)`?

Reply:
I usually don't like doing that, since it implies that the framework we're building Roslyn and the frameworks Roslyn supports running against are the same. It's moot in this case, but we do see places where people think "I have to upgrade what I build my analyzer with" so they can use nameof, even though it's the wrong thing to do.